fluxion/decoders/gstreamer.py

```diff
@@ -62,6 +62,24 @@ class GStreamer010(GStreamer):
     gst_api = "0.10"
 
 
+@register_decoder
+class GStreamerVaapiH265Gst10Decoder(GStreamer10):
+    '''GStreamer H.265 VAAPI decoder implementation for GStreamer 1.0'''
+    codec = Codec.H265
+    decoder_bin = ' h265parse ! vaapih265dec '
+    caps = 'video/x-raw,format=I420'
+    api = "VA-API"
```

Comment:
provider = "GStreamer" is missing

Reply:
I'll add that to the base GStreamer classes so that we don't forget it